integration/v3_grpc_inflight_test.go

```diff
@@ -54,6 +55,101 @@ func TestV3MaintenanceDefragmentInflightRange(t *testing.T) {
 	<-donec
 }
 
+// TestV3CompactInflightHashKV ensures that HashKV returns
+// correct hash while compacting.
+func TestV3CompactInflightHashKV(t *testing.T) {
```

Comment:
i saw that mvcc pkg only have unit tests. In that case, should i just movie the exact same test to some new test.go file in mvcc? or should I transform my test to some sort of unit test on kvstore?